api/layer/cors.go

```diff
@@ -37,10 +37,11 @@ func (n *layer) PutBucketCORS(ctx context.Context, p *PutCORSParams) error {
 	}
 
 	prm := PrmObjectCreate{
-		Container: p.BktInfo.CID,
-		Creator:   p.BktInfo.Owner,
-		Payload:   p.Reader,
-		Filename:  p.BktInfo.CORSObjectName(),
+		Container:    p.BktInfo.CID,
+		Creator:      p.BktInfo.Owner,
+		Payload:      p.Reader,
+		Filename:     p.BktInfo.CORSObjectName(),
+		CopiesNumber: 0,
```

Comment:
Why `0`? It should be value from config

Reply:
Ok, I thought we should configure `CopiesNumber` only for common obejcts. I will fix it.